shinysdr/plugins/multimon.py

```diff
@@ -253,7 +253,7 @@ def __lineReceived(self, line):
 
 
 # TODO: Arrange for a way for the user to see why it is unavailable.
-_multimon_available = test_subprocess('multimon-ng -h; exit 0', 'available demodulators:', shell=True)
+_multimon_available = test_subprocess('multimon-ng -h; exit 0', 'Available demodulators:', shell=True)
```

Comment:
Because the change in multimon-ng is so recent, please change the substring to `'vailable demodulators:'` for now.